src/Parser/BalanceDruid/Modules/Spells/UnempoweredLs.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import Module from 'Parser/Core/Module';
+import SPELLS from 'common/SPELLS';
+import SpellLink from 'common/SpellLink';
+import { formatNumber } from 'common/format';
+
+class UnempoweredLS extends Module {
+  _castQueue = {
+      Empowered: false,
+      Enemies: 0,
+    };
+  casts = [];
+  _lunarEmpsOn = false;
+
+  suboptUmempLS = 0;
+
+  isLunarStrike(event) {
+    const spellId = event.ability.guid;
+    return spellId === SPELLS.LUNAR_STRIKE.id;
+  }
+  isLunarEmpowerment(event){
+      const spellId = event.ability.guid;
+      return spellId === SPELLS.LUNAR_EMP_BUFF.id;
+  }
+
+  on_byPlayer_cast(event) {
+    if (this.isLunarStrike(event)) {
+        this.casts.push(this._castQueue);
+        this._castQueue = {
+            Empowered: this._lunarEmpsOn,
+            Enemies: 0,
+        };
+    }
+  }
+
+  on_byPlayer_damage(event) {
+    if (!this.isLunarStrike(event)) {
+      return;
+    }
+    this._castQueue.Enemies++;
+  }
+
+  on_toPlayer_applybuff(event){
+    if (this.isLunarEmpowerment(event)) {
+        this._lunarEmpsOn = true;
+    }
+  }
+  on_toPlayer_removebuff(event){
+    if (this.isLunarEmpowerment(event)) {
+        this._lunarEmpsOn = false;
+    }      
+  }
+
+  on_finished() {
+    this.casts.push(this._castQueue);
+
+    this.suboptUmempLS = this.casts.filter(function(cast){
```

Comment:
This could be a one liner; `cast => !cast.Empowered && cast.Enemies > 0 && cast.Enemies < 3`